apps/yggdrasil/test/test_helper.exs

```diff
@@ -1,4 +1,9 @@
 ExUnit.start()
 
-Mix.Task.run "ecto.create", ~w(-r Yggdrasil.Repo --quiet)
-Mix.Task.run "ecto.migrate", ~w(-r Yggdrasil.Repo --quiet)
+repo = ~w(-r Yggdrasil.Repo --quiet)
+
+Mix.Task.run "ecto.create", repo
+Mix.Task.run "ecto.migrate", repo
+
+run_path = Application.app_dir :yggdrasil, Path.join(["priv", "repo", "seeds.exs"])
+Mix.Task.run "run", [run_path]
```

Comment:
@copenhas This does work, when I first attempted this I had a few issues like `seed.ex`, `seeds.ex` when it should have been `seeds.ex`.

All that aside, I still kept getting an error and the thing that through me off is that it wasn't clear that I needed `[run_path]`. I was defining it like `"run", [run_path]`. It was so confusing, but apparently ~w(...) returns an array which is obvious now that I think about it. haha.

`Application.app_dir/2` does return the app path in the _build directory but sym links to priv in the same structure os the apps/yggdrasil dir so that's all that is needed.